packages/preset-mini/src/rules/position.ts

```diff
@@ -96,7 +96,7 @@ function handleInsetValues([, d, v]: string[]): CSSEntries | undefined {
 }
 
 export const insets: Rule[] = [
-  [/^(?:position-|pos-)?inset-()(.+)$/, handleInsetValues],
+  [/^(?:position-|pos-)?inset-()(.+)$/, ([, , v]) => ({ inset: handleInsetValue(v) })],
```

Comment:
The empty capture `()` can be removed.

Reply:
Alright, I'm new to this. Thanks for figuring it out